changelogs/fragments/39-add_folder_template.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Does it support storing it only in a folder?

Reply:
Yes, I think the other option is storing it in a content library. Even if the user didnt specify a folder path, it would technically be stored in the `/<datacenter>/vms` folder

Reply:
"Normal" templates are quite old, content libraries are much younger. AFAIK there are significant differences between them, both technically and how people use them.